### DISCUSSION TITLE
Pick secret of my pocketbase application

### DISCUSSION THREAD

**Discussion by lokize** [pocketbase/pocketbase#784] [Q&A] [answered]
hello friends, I need to decode the token in my backend to verify the authenticity of the tokens generated by pocketbase, how do I know the secret of my application?

**Comment by ganigeorgiev** [comment, marked as answer]
The secret of each token is constructed by 2 parts:

- unique `tokenKey` stored alongside each user in the `_users` table
- common secret that you can access from the app settings based on the token type (auth, verification, password reset, etc.)

The easiest way to verify a PocketBase auth token would be to send `POST /api/users/refresh` API request with `Authorization: User YOUR_TOKEN` header - https://pocketbase.io/docs/api-users/.

**Reply by lokize** [reply]
thank you!
